docs/workshops-materials.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Workshops' materials
+date: 2024-03-21
+sidebar_position: 13
+---
+
+# Workshops' materials
+
+Below you can find materials we have prepared for our workshops.
+
+## Packit downstream automation workshops
+
+1) Fedora Release Guide *for beginners!*
+
+   [**slides**](https://docs.google.com/presentation/d/e/2PACX-1vRjHzcBoiW7vOq4lADmN5LLbnwlBRM3QsFp2-1u5en0wWhImq03ASI0D12vw_chPAouh_AdL-GttnVW/embed?slide=id.p)
+   
+   <video controls>
+     <source src="https://www.youtube.com/watch?v=Wh07Sfu0xQE"/>
+   </video>
```

Comment:
Not sure if this will work. Visual Studio Code Preview is showing the video embedded, but github is showing nothing. I would just try to merge it and see if it working for real.

Reply:
Lemme try locally

Reply:
No, it doesn't. If you open the video in a browser and click and “Share”, there is an option for “embed”. (I also ticked the privacy thingy) That works for me (example):

```md
1) Fedora Release Guide *for beginners!*

   [**slides**](https://docs.google.com/presentation/d/e/2PACX-1vRjHzcBoiW7vOq4lADmN5LLbnwlBRM3QsFp2-1u5en0wWhImq03ASI0D12vw_chPAouh_AdL-GttnVW/embed?slide=id.p)
   
   <iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/Wh07Sfu0xQE?si=WJdlT5qec38_jeZj" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
```

![image](https://github.com/packit/packit.dev/assets/8149784/5cc2b676-7b6d-45bb-a167-e1e185fe8b8a)

Reply:
Ok! Thanks for looking at it!